src/test/java/com/epam/tc/hw3/ex2/SecondExercise.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.tc.hw3.ex2;
+
+import com.epam.tc.hw3.BaseClass;
+import com.epam.tc.hw3.pages.DifferentElementsPage;
+import com.epam.tc.hw3.pages.IndexPage;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class SecondExercise extends BaseClass {
+
+    SoftAssert softAssert = new SoftAssert();
+
+    @Test
+    public void secondTest() {
```

Comment:
Please rename test method with some meaningful name.
Test methods should have a name in accordance with checking functionality

Reply:
Renamed